WcaOnRails/spec/controllers/delegate_reports_controller_spec.rb

```diff
@@ -54,7 +54,7 @@
       expect(comp.delegate_report.remarks).to eq "My new remarks"
     end
 
-    it "can edit report before comp is over, but cannot post report" do
+    it "can edit report before comp is over, and can post report" do
```

Comment:
Is the comp really "probably over" here, since we fudge it by one day in the code?

Regardless: `expect(comp.is_probably_over?).to eq false'

Also: `"can edit and post report before comp is over"`